app/controllers/projects_controller.rb

```diff
@@ -20,34 +17,38 @@ def new
   end
   
   def create
-    @project = Project.new(project_params)
-    @project.user = current_user # Assuming you're associating the project with the current user
+    @project = Project.new(project_params) if current_user.manager?
 
     if @project.save
-      redirect_to @projects, notice: 'Project was successfully created.'
+      redirect_to projects_path, notice: ('flash.project.created')
     else
       render :new
     end
   end
 
   def edit
     @project = Project.find(params[:id])
```

Comment:
`@project = Project.find(params[:id])` this is repeated in multiple actions, we can use callback and private method to set the @project